lib/tartan/pallet.py

```diff
@@ -0,0 +1,243 @@
+# Authors: see git history
+#
+# Copyright (c) 2023 Authors
+# Licensed under the GNU GPL version 3.0 or later.  See the file LICENSE for details.
+# Additional credits to: https://github.com/clsn/pyTartan
+
+import re
+from typing import List
+
+import wx
+from inkex import Color
+
+from .colors import string_to_color
+
+
+class Pallet:
```

Comment:
English nit: we want to spell this as "palette", otherwise we're talking about a bed :)

Reply:
Oh no. You read my real thoughts while writing that, let's wake up then :)